packages/zowe-explorer/__tests__/__integration__/bdd/features/dialogs/IssueCommand.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Issue TSO Command
+
+Scenario Outline: User wants to issue a command
+    Given a user who is looking at the Zowe Explorer tree views
+    When a user selects <opt> from the command palette
+    Then a quick pick appears to select a profile
+    When a user selects a profile
+    Then a user can enter in <cmd> as the command and submit it
+
+    Examples:
+      | opt | cmd |
+      | Issue TSO Command | /profile |
+      | Issue MVS Command | /D T |
```

Comment:
Curious why there is a slash in front of these 2, since MVS and TSO commands aren't typically formatted that way?

Reply:
To be honest, I took the commands right from the old "Issue command" integration tests, so it's possible that those were not valid commands to test with. Looking back, the response of the API was spoofed at the time, so I don't think the command had any effect.

Since the integration config points to `localhost`, the commands won't be sent in this scenario either - but I will make some adjustments to use valid commands for the test 😋